impeller/typographer/font.h

```diff
@@ -50,7 +50,7 @@ class Font : public Comparable<Font> {
 
   Font(std::shared_ptr<Typeface> typeface, Metrics metrics);
 
-  ~Font();
+  virtual ~Font();
```

Comment:
also question here: why do we need a virtual destructor?

Reply:
Yep, because we have virtual methods (via inheritance).

Reply:
but this isn't being implemented anywhere

Reply:
You are right.  It's my understanding that this should be fine.  The reason I added it was that when using `std::shared_ptr<Font>` it complained about it not being virtual.  I'm not sure why.  I would have only expected that to be a problem if we are calling delete on a `Comparable<Font>*`.  I'll remove this.

Reply:
done